cli/src/commands/util/gc.rs

```diff
@@ -24,6 +24,9 @@ use crate::command_error::CommandError;
 use crate::ui::Ui;
 
 /// Run backend-dependent garbage collection.
+///
+/// To abandon old operations, run `jj op abandon ..<some old operation>` before
+/// `jj util gc`.
```

Comment:
nit: Should we say "To garbage-collect old operations, ..." instead? Because otherwise the "before ..." part isn't needed. Or maybe it's already clear and I'm just being overly pedantic (very possible).

Reply:
Or "To garbage-collect old hidden commits"? If the user knows reachable commits cannot be GC-ed, they would do `jj op abandon` without a hint.

Reply:
Good point! How about this?

Reply:
Looks good, thanks.